src/sqlfluff/utils/testing/cli.py

```diff
@@ -0,0 +1,29 @@
+"""Testing utils for working with the CLIs."""
+from click.testing import CliRunner
+
+
+def invoke_assert_code(
```

Comment:
Moved this function here from `test/cli/commands_test.py` so it can be used by `sqlfluff-templater-dbt` tests.